src/main/resources/gather-hosts.yml

```diff
@@ -1,9 +1,23 @@
 ---
 - hosts: all
+
   vars:
     facts: True
-  gather_facts: "{{facts}}"
+
+  gather_facts: "{{ facts }}"
+
   tasks:
-    - local_action: file path="{{tmpdir}}/data" state=directory
+    - name: "Ensure tmpdir data directory"
+      file:
+        path: "{{ tmpdir }}/data"
+        state: directory
+      become: no
       run_once: yes
-    - local_action: template src=host-tpl.j2 dest="{{tmpdir}}/data/{{inventory_hostname}}"
+      delegate_to: localhost
+
+    - name: "Template the gathered facts"
+      template:
+        src: host-tpl.j2
+        dest: "{{ tmpdir }}/data/{{ inventory_hostname }"
```

Comment:
@fboux0 @frozenice there is a syntax error leading to 

fatal: [db266ac2.d-w.wuerth.com]: FAILED! => {"msg": "template error while templating string: unexpected '}'. String: {{ tmpdir }}/data/{{ inventory_hostname **}"}**

It hould be 
```
dest: "{{ tmpdir }}/data/{{ inventory_hostname }}"
```

Reply:
i reverted it for now to keep the master branch working.

Reply:
ah nevermind i just put up a fix PR #227 instead as the revert also requires a review :)

Reply:
Ah my apologies, it was my first push to GitHub and I must have missed my final local commit. Fix looks good.